recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+
+test:
+  commands:
+    - test -d $PREFIX/include/blitz  # [unix]
```

Comment:
Could we please space out this selector so it is inline with the other selectors below?